src/actions/guides/database/migrations.cr

```diff
@@ -254,15 +254,15 @@ class Guides::Database::Migrations < GuideAction
 
     > You can also use a symbol for a table name. For example `create :users`.
 
-    ## Using fill_existing_with
+    ## Using fill_existing_with and default values
 
-    When using the `add` method inside an `alter` block, there's an additional option `fill_existing_with`.
+    When using the `add` method inside an `alter` block, there's an additional option `fill_existing_with`.  This is useful when a default value is **NOT** needed, but the existing data should be backfilled anyway.
```

Comment:
```suggestion
    When using the `add` method inside an `alter` block, there's an additional option `fill_existing_with`.  This is useful when a default value is **NOT** desired, but the existing data should be backfilled anyway.
```

Reply:
Just changed to `desired` since it is more about what you want to do. You could do either default or fill_existing_with depending on what you want. `fill_existing_with` is usually used when you don't want a default. You want ever new record to specify a value for the column. `default` is better when you have a sensible default (like for an `admin` column with a default of `false`)